README.md

```diff
@@ -14,8 +15,72 @@ ember install ember-power-select-infinity
 Usage
 ------------------------------------------------------------------------------
 
-[Longer description of how to use the addon in apps.]
+To use `power-select-infinity` you just declare it in a template.
+Both `search` and `loadMore` must return a promise to that `power-select-infinity` can display the loading indicator.
+The paging is left up to you, since some API's use different formats for paging. Some API's declare page=1, others take in the last items ID.
+Either way, `power-select-infinity` just provides a `loadMore` action where you can load data however your API requires.
 
+Your Component
+```
+{{#power-select-infinity
+    search=(action 'search')
+    loadMore=(action 'loadMore')
```

Comment:
@bakerac4  Will this also support the `options` attribute in addition to the `search` action like the normal power-select does? per the docs:
> You can provide both options and a search action. Those options will be the initial set of options, but as soon as the user performs a search, the results of that search will be displayed instead.

I'd say thats definitely a pretty common thing to need, at least in apps I've done, where you show the user the first 10 or so options which are preloaded when coming to the page.

Reply:
@billdami Yes, looking at the examples thats how you define the initial list